prusti-viper/src/encoder/vir/cfg/method.rs

```diff
@@ -12,30 +12,35 @@ use uuid::Uuid;
 
 pub(super) const RETURN_LABEL: &str = "end_of_method";
 
-#[derive(Debug, Clone)]
-pub struct CfgMethod {
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct CfgMethod { // TODO: Should labels and reserved_labels be ignored?
```

Comment:
Regarding the TODO: the best thing here would be to
1. rename `CfgMethod` to `CfgMethodBuilder`, and remove the serialize/deserialize from it
2. add a new, much simpler, `CfgMethod` (serializable, without `labels`, `reserved_labels`, ...)
3. add a `fn build(self) -> CfgMethod` to `CfgMethodBuilder`
4. move any `to_viper` method from `CfgMethodBuilder` to `CfgMethod`

If we don't do this, then skipping label fields is ok as a temporary solution (if they are not used in the conversion to Viper).